File: src/main/java/com/github/wgzhao/dbquery/repo/QueryConfigSignRepo.java
package com.github.wgzhao.dbquery.repo;

import com.github.wgzhao.dbquery.entities.QueryConfigSign;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface QueryConfigSignRepo extends JpaRepository<QueryConfigSign, Long> {
    boolean existsByAppId(String AppId);

    List<QueryConfigSign> findBySelectId(String queryConfigId);

    List<QueryConfigSign> findByAppId(String AppId);

    boolean existsBySelectIdAndAppId(String queryConfigId, String AppId);

    QueryConfigSign findBySelectIdAndAppId(String queryConfigId, String AppId);

    void deleteByAppId(String appId);
}

File: src/main/kotlin/com/github/wgzhao/dbquery/repo/QueryConfigSignRepo.kt
package com.github.wgzhao.dbquery.repo

import com.github.wgzhao.dbquery.entities.QueryConfigSign
import org.springframework.data.jpa.repository.JpaRepository

interface QueryConfigSignRepo : JpaRepository<QueryConfigSign, Long> {
    fun existsByAppId(appId: String): Boolean

    fun findBySelectId(queryConfigId: String?): MutableList<QueryConfigSign?>?

    fun findByAppId(appId: String): List<QueryConfigSign?>?

    fun existsBySelectIdAndAppId(queryConfigId: String?, appId: String?): Boolean

    fun findBySelectIdAndAppId(queryConfigId: String?, appId: String?): QueryConfigSign?

    fun deleteByAppId(appId: String?)
}